app/default/content_routes.py

```diff
@@ -68,19 +70,22 @@ def cookie_policy():
 @content_bp.route("/privacy", methods=["GET"])
 def privacy():
     current_app.logger.info("Privacy_notice page loaded.")
-    round_data = get_round_data_fail_gracefully(
-        Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID
-    )
+    fund_short_name = request.args.get("fund")
+    round_short_name = request.args.get("round")
+    if fund_short_name and round_short_name:
+        round_data = get_round_data_by_short_names(
+            fund_short_name, round_short_name
+        )
+        privacy_notice_url = getattr(round_data, "privacy_notice", None)
 
-    privacy_notice_url = getattr(round_data, "privacy_notice", None)
+        if privacy_notice_url:
+            current_app.logger.info("Privacy notice configured for fund")
+            return redirect(privacy_notice_url)
 
-    if privacy_notice_url:
-        current_app.logger.warning("Privacy notice configured for fund")
-        return redirect(privacy_notice_url)
-    else:
-        current_app.logger.warning(
-            "No privacy notice configured for fund. Redirecting..."
-        )
-        return redirect(
-            "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice"  # noqa
-        )
+    current_app.logger.warning(
+        f"No privacy notice configured for round ({fund_short_name} -"
+        f" {round_short_name}). Redirecting..."
+    )
+    return redirect(
+        "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice"  # noqa
```

Comment:
If there's no privacy notice configured, should COF be the default or a404?

Reply:
creating another ticket to adjust this